client/verta/verta/client.py

```diff
@@ -863,8 +863,8 @@ def get_or_create_endpoint(self, path=None, description=None, workspace=None, pu
             return Endpoint._get_by_id(self._conn, self._conf, workspace, id)
         else:
             return Endpoint._get_or_create_by_name(self._conn, path,
-                                            lambda path: Endpoint._get_by_path(self._conn, self._conf, workspace, path),
-                                            lambda path: Endpoint._create(self._conn, self._conf, workspace, public_within_org, path, description),
+                                            lambda name: Endpoint._get_by_path(self._conn, self._conf, workspace, path),
```

Comment:
Keep it for consistency

Reply:
Huh, I don't even know how that change made it in here; it's totally unrelated 🤔